apps/meteor/client/sidebar/header/UserDropdown.tsx

```diff
@@ -46,7 +46,7 @@ const setStatus = (status: typeof userStatus.list['']): void => {
 	callbacks.run('userStatusManuallySet', status);
 };
 
-const getItems = (): ReturnType<typeof AccountBox.getItems> => AccountBox.getItems();
+const getItems = (): IAppAccountBoxItem[] => AccountBox.getItems() as IAppAccountBoxItem[];
```

Comment:
I don't think you need to use `as` here, since `getItems` will return an array of `(IAppAccountBoxItem | AccountBoxItem)` and afterwards you will filter this array to get only the apps Items on `appBoxItems`. 

I would suggest keeping with the typing of `IAppAccountBoxItem | AccountBoxItem` for the `getItems`. If needed a type guard can be done in `appBoxItems` to stop errors

Reply:
Done!